compiler/x/codegen/OMRX86Instruction.cpp

```diff
@@ -1153,7 +1153,11 @@ void
 TR::X86RegImmSymInstruction::autoSetReloKind()
    {
    TR::Symbol *symbol = getSymbolReference()->getSymbol();
-   if (symbol->isConst())
+   if (getSymbolReference()->isDebugCounter())
```

Comment:
minor nit, since nobody else who changed this code really thought about it, but putting this relatively rare case condition at the beginning if the if chain isn't the best performance choice :) .